src/libs/firebase/firestore/featureFlags/useFeatureFlag.native.test.ts

```diff
@@ -64,4 +65,38 @@ describe.each([
       expect(result.current).toBe(firebaseFeatureFlag)
     }
   )
+
+  it('should deactivate FF when no build number is given', async () => {
```

Comment:
les `it` qui suivent devraient probablement etre dans un autre `describe`

on avait que les tests actuels étaient améliorables, on a choisi de ne pas mettre le focus dessus dans un premier temps

itérer sur les FF pour ces tests n'apporte rien

Reply:
J'ai ajouté un commit pour refacto les tests